src/main/java/no/ssb/dapla/migration/agent/cmd/SasAgentClientImpl.java

```
@@ -197,11 +195,22 @@ public SasAgentClientImpl(Config config, JwtVerifier jwtVerifier) {
         if (!Files.isWritable(signatureFolder)) {
             throw new RuntimeException("Signature Folder is NOT writable: " + signatureDataFolder);
         }
+        //Stop copy job if disk space is less than 1 GB
+        if(DiskSpaceUtil.getUsableSpace()<1024*1024*1024) {
```

I agree with what Magnus said about this in the meeting today: We should avoid the "magic number" approach of returning an error when there is less than 1G left. Instead, we should return an error if there is less space left than the size of the file we are copying.